specification/visualstudio/resource-manager/Microsoft.VisualStudio/2014-04-01-preview/examples/CreateProjectResource.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "parameters": {
+    "body": {
+        "id": "/subscriptions/0de7f055-dbea-498d-8e9e-da287eedca90/resourceGroups/VS-Example-Group/providers/Microsoft.VisualStudio/account/ExampleAccount/project/ExampleProject",
+        "name": "ExampleProject",
+        "type": "Microsoft.VisualStudio/account/project",
+        "location": "Central US",
+        "tags": {},
+        "properties": {
+          "ProcessTemplateId": "6B724908-EF14-45CF-84F8-768B5384DA45",
```

Comment:
I am not sure how best to express the fact that these two properties are required (ProcessTemplateId and VersionControlOption).

Also, there's a relatively static range of values that are accepted for these. If they were formal parameters I'd consider using x-ms-enum in the RP spec to list all possible values, but in this particular case I am not sure how to express that in documentation.